### ISSUE TITLE
Packet analyzer

### ISSUE THREAD

**Issue by djcalcutt** [JMRI/website#546]
I don't think I did this right but I'm gonna be in the wrong section using an iOS application to file an issue

Using original pack analyzer and JMR version 5.9.1 

When clicking on the idle packets bullet, cause I don't wanna see the idle packets, the idle packets still show   

Also, when cooking the checkbox to reduce duplication, Carriage returns are not displayed so the line continues 

![image](https://github.com/JMRI/website/assets/61532140/e619768d-f151-40c4-a0ed-d83926deb8f7)

In my live video feed the packet analyzer data is displayed so you can see this for yourself at 
https://www.youtube.com/live/h19GOt1TmII?si=oGb66L5CofMPdmpi

It's a great improvement. Thanks for the great work.  just use the packet analyzer to verify speed steps for people who operate trains online

**Comment by KenC57** [comment]
I'll look into this but I this issue is showing on the website instead
of the normal code for some reason.

**Comment by djcalcutt** [comment]
Yeah, sorry that’s my mistake. I was posting it with an iOS application and I couldn’t seem to find the proper place to post it.  Darrin J. CalcuttSent from my iPhoneOn Jul 1, 2024, at 14:39, Ken Cameron ***@***.***> wrote:﻿
I'll look into this but I this issue is showing on the website instead
of the normal code for some reason.

**Comment by djcalcutt** [comment]
Here is the funny part. 

I have two identical computers, both with windows 10, both using 5.9.1 jmri

Both are using the same com port (serial server. 

Yet this computer using the dcc packet analyzer displays properly. 

Now this is puzzling. ![image](https://github.com/JMRI/website/assets/61532140/152a5c04-2d1c-4654-8231-c4709ac03127)

**Comment by KenC57** [comment]
DJ,

Same version of JMRI,
Same version of Windows.
Same Original packet analyzer.
Right?
Check if the Preferences->Display preferred appearance is the same.
Like Metal, Windows Classic, Nimbus, etc.

**Comment by djcalcutt** [comment]
Thank you. Darrin J. CalcuttSent from my iPhoneOn Jul 4, 2024, at 20:12, Ken Cameron ***@***.***> wrote:﻿
DJ,

Same version of JMRI,
Same version of Windows.
Same Original packet analyzer.
Right?
Check if the Preferences->Display preferred appearance is the same.
Like Metal, Windows Classic, Nimbus, etc.

**Comment by djcalcutt** [comment]
my packet analyzer died. So untill i get it replaced... you can close this ticket.
